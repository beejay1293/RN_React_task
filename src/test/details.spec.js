import React from 'react';
import { shallow } from 'enzyme';
import  Details from '../components/Dashboard/details';
import store from '../stores';

describe('test Dashboard component', () => {
  it('should render component', () => {
    const Det = shallow(<Details  store={store}/>);
    expect(Det).toMatchSnapshot();
  });
});
