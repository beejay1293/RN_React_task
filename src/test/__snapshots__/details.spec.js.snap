// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Dashboard component should render component 1`] = `
<ContextProvider
  value={null}
>
  <withRouter(Detail)
    details={Array []}
    getDetails={[Function]}
    loading={false}
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  />
</ContextProvider>
`;
